scripts/maintenance/virsh-cleanup.sh

```diff
@@ -23,13 +22,22 @@ do
 	run virsh -c "${CONNECT}" undefine "${DOMAIN}"
 done
 
-virsh -c "${CONNECT}" vol-list "${POOL}" | tail -n +3 | while read -r VOLUME _
+for POOL in $(virsh -c "${CONNECT}" pool-list --all --name)
 do
-	if test -z "${VOLUME}"
-	then
-		continue
-	fi
-	run virsh -c "${CONNECT}" vol-delete --pool "${POOL}" "${VOLUME}"
+        if test "${POOL}" = default
+        then
+                continue
+        fi
```

Comment:
First time I look at this script, but it seems destroying all volumes including the ones in the default pool would be more consistent with what it's doing (which is removing all libvirt vms/net/...). Apart from this, looks good to me.